src/tests/ftest/scrubber/aggregation.yaml

```diff
@@ -39,20 +39,16 @@ pool:
 container:
   type: POSIX
   control_method: daos
-  oclass: RP_2GX
   properties: "cksum:sha512,rd_fac:1"
 ior:
   ior_timeout: 60
-  client_processes:
-    np: 6
```

Comment:
Why not use multiple procs to make this faster?

Reply:
I can give it a try... I was playing around with the delayed checksum corrupt mechanism and removed the multiple procs. Let me re-enable and see how it goes...

Reply:
Using any value other than 1 results in the first IOR run to fail with fault getting injected. I am going to keep the process as 1 to make it simple for now (eventhough the ior runs make take sometime).